packages/atlas/src/components/ListItem/ListItem.tsx

```diff
@@ -58,50 +60,65 @@ export const ListItem = forwardRef<HTMLDivElement, ListItemProps>(
       highlightWhenActive,
       to,
       externalLink,
+      isSeparator,
     },
     ref
   ) => {
     const [hoverRef, isHovering] = useHover<HTMLDivElement>()
     const linkProps = getLinkPropsFromTo(to)
+
+    const labelColor = useMemo(() => {
+      if (destructive) {
+        return 'colorTextError'
+      }
+      if (isSeparator) {
+        return 'colorText'
```

Comment:
It should be `colorTextMuted` If I'm not mistaken. 
![image](https://user-images.githubusercontent.com/51168865/197518551-d24f9967-d856-43b2-9d75-340db1342005.png)